### ISSUE TITLE
Missing Data

### ISSUE THREAD

**Issue by EasternCaveMan** [AlexanderKroll/ESP#15]
Dear Authors,
would you please provide the missing files:
`/ESP/data/enzyme_data/Uniprot_df_with_ESM1b.pkl`
`/ESP/data/splits/df_train.pkl`
`/ESP/data/splits/df_test.pkl`
Best,
Vahid

**Comment by AlexanderKroll** [comment]
Dear Vahid,

the files are not included in the repo because they should be easily reproducible from the uploaded files. For example, "df_train.pkl" and "df_test.pkl" are just slight modifications of "df_UID_MID_train_phylo.pkl", "df_UID_MID_train_exp.pkl" and 
"df_UID_MID_test_exp_phylo.pkl", all of which are uploaded (see https://github.com/AlexanderKroll/ESP/blob/main/notebooks_and_code/1_0%20-%20Creating%20enzyme-substrate%20database%20from%20GOA%20database.ipynb). 

For "/ESP/data/enzyme_data/Uniprot_df_with_ESM1b.pkl" you would have to calculate the ESM-1b representations for all sequences using the fair-esm python package. If you have trouble doing this, I have uploaded this file for you here: https://drive.google.com/file/d/1BMMhgEQ0ILWoVJKEJLRVyLfxDmqYR9nW/view?usp=sharing

I hope this helps you to get/reproduce the missing files.

Good luck with your thesis! Best,
Alex

**Comment by AlexanderKroll** [comment]
Dear Vahid, 

I used the all sequences (including the ones with phylogenetic evidence) to ensure that the phylogenetic data in the training set is not too similar to the test set. If you have problems running the cd-hit algorithm, you could send me a sequences.fasta file and I can calculate the output of the cd-hit algorithm for you. In this case you could also set the IDs in a different way (I agree that how I did it was unfortunate).

Best,
Alex
